build.gradle

```diff
@@ -141,7 +141,7 @@ ext {
         mockito: 'org.mockito:mockito-core:3.9.0',
         mockk: 'io.mockk:mockk:1.12.3',
         mockkandroid: 'io.mockk:mockk-android:1.12.3',
-        mockwebserver: 'com.squareup.okhttp3:mockwebserver:4.9.1',
+        mockwebserver: 'com.squareup.okhttp3:mockwebserver:5.0.0-alpha.9',
```

Comment:
why are we using external alpha release?

Reply:
AWS Kotlin SDK is using alpha release which is causing build failures due to dependency version conflicts.